app/pages/lab/visibility.cjsx

```diff
@@ -61,7 +61,7 @@ module.exports = React.createClass
       ).then(() => @forceUpdate())
 
   handleWorkflowStatsVisibility: (workflow, e) ->
-    checked = e.target.checked
+    checked = !e.target.checked
```

Comment:
This is very confusing. For clarity, I would name this variable `hidden` or something similar.